src/CLI/L1AddressOrAlias.hs

```diff
@@ -4,122 +4,16 @@
 {-# OPTIONS_GHC -Wno-orphans #-}
 
 -- TODO: Remove this module when upgrading to the next morley release.
```

Comment:
Well, this ended up getting quite small here, why not solve this TODO and actually remove it?

Reply:
I was thinking we should probably move this instance upstream (didn't yet create an issue, but I will do that now). Might as well move the instance to some other module, but if we're going to remove it, arguably it wouldn't make a huge difference.

Reply:
Oh, no, actually I did: https://gitlab.com/morley-framework/morley/-/issues/923, just forgot about it :sweat_smile:

Reply:
One reason to keep it in a separate module is `-Wno-orphans`. Semantically, it should live either in `Client.Parser` or `CLI.Parser`, but neither of those has `-Wno-orphans` and adding it to a larger module is kind of meh.

Reply:
ah, I see, sounds good to me :ok_hand: 

In that case please mention `morley/#923` in this TODO for the next time instead :slightly_smiling_face: